Example/ViewController.swift

```diff
@@ -666,6 +666,7 @@ class ViewController: UIViewController {
             self.navigationMapView = nil
             self.passiveLocationManager = nil
             navigationViewController.navigationMapView?.showsRestrictedAreasOnRoute = true
+            navigationViewController.navigationMapView?.showsIntersectionSignals = true
```

Comment:
This only gets called when starting the trip, but not when the style manager automatically switches between DayStyle and NightStyle. A style image that you add to the map’s day style will go away when the map changes to the night style.

`NavigationViewController.updateMapStyle(_:)` should be responsible for calling `updateIntersectionSymbolImages()` and maybe `removeIntersectionSignals()`. This would also be a natural place to get the style images from the current `Style`.

Reply:
The style change of the icons has been handled by `NavigationViewController` and `CarPlayNavigationViewController` after the style applied by `StyleManager`, instead of the `NavigationMapView.styleType`, which is removed